.github/workflows/import_pending.yml

```diff
@@ -14,21 +14,29 @@ jobs:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
-      - name: Extract branch name
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
-        id: extract_branch
+
+      - uses: denoland/setup-deno@v1
+        with:
+          # Specifies latest 1.x
+          deno-version: "~1.0"
+
       - name: Import and commit pending sql
         run: |
           git config user.email "azerothcorebot@gmail.com" && git config user.name "AzerothCoreBot"
           bash bin/acore-db-pendings
-          bash bin/acore-import-changelogs
-          git fetch --unshallow origin ${BRANCH}
+          deno apps/ci/ci-pending-changelogs.ts
```

Comment:
Just help me understand why we are changing pending changelog and not pending db?

Reply:
`bin/acore-db-pendings` is a shell script, and doesnt have any typescript to be used with deno. After it starts, it sources another shell script (`apps/ci/ci-pending-sql.sh`) and doesn't use any typescript

`bin/acore-import-changelogs` sources `apps/bash_shared/includes.sh`. This installs Deno. It then executes the typescript file at `apps/ci/ci-pending-changelogs.ts`. Proxying the typescript file through the `acore-import-changelogs` shell script doesn't seem to add anything and it makes the execution a bit harder to trace. having the CI directly run `deno apps/ci/ci-pending-changelogs.ts` removes an unnecessary abstraction.

Reply:
kk